setup.py

```diff
@@ -1,13 +1,25 @@
 from setuptools import setup, find_packages
 
+import minid_client
+
+install_requires = []
+with open('requirements.txt') as reqs:
+    for line in reqs.readlines():
+        req = line.strip()
+        if not req or req.startswith('#'):
+            continue
+        install_requires.append(req)
+
+
 setup(
     name='minid',
-    version='1.1',
+    version=minid_client.__VERSION__,
```

Comment:
The decision here to remove version from the `setup.py` file is so that it can be used outside of the `setup.py` file. This isn't strictly necessary for packaging, but it's a little more standard and would be useful for providing a REST `/version` endpoint for clients in the future to check of the server they're connecting to is compatible.